cmd/agent/ngt/main.go

```diff
@@ -46,12 +46,12 @@ func main() {
 				"cgo enabled": info.CGOEnabled,
 				"ngt version": info.NGTVersion,
 			})),
-			runner.WithConfigLoader(func(path string) (interface{}, string, string, error) {
+			runner.WithConfigLoader(func(path string) (interface{}, config.Default, error) {
 				cfg, err := config.NewConfig(path)
 				if err != nil {
-					return nil, "", "", err
+					return nil, config.Default{}, err
```

Comment:
![badge](https://img.shields.io/badge/review-suggestion-blue.svg?style=flat-square)
this should be renamed as `config.Common`

Reply:
fixed